server/webapp/WEB-INF/rails.new/spec/presenters/api_v3/plugin/authentication_plugin_info_representer_spec.rb

```diff
@@ -29,36 +29,11 @@
     plugin_info = com.thoughtworks.go.plugin.domain.authentication.AuthenticationPluginInfo.new(descriptor, "display_name", 'display_image_url', true, true, plugin_settings)
     actual_json = ApiV3::Plugin::AuthenticationPluginInfoRepresenter.new(plugin_info).to_hash(url_builder: UrlBuilder.new)
 
-    expect(actual_json).to have_links(:self, :doc, :find)
-    expect(actual_json).to have_link(:self).with_url('http://test.host/api/admin/plugin_info/foo.example')
-    expect(actual_json).to have_link(:doc).with_url('https://api.gocd.org/#plugin-info')
-    expect(actual_json).to have_link(:find).with_url('http://test.host/api/admin/plugin_info/:plugin_id')
-    actual_json.delete(:_links)
-
     expect(actual_json).to eq({
-                                id: 'foo.example',
-                                version: '1.0',
-                                type: 'authentication',
-                                status: {
-                                  state: 'active'
-                                },
-                                plugin_file_location: nil,
-                                bundled_plugin: false,
-                                about: {
-                                  name: 'Foo plugin',
-                                  version: '1.2.3',
-                                  target_go_version: '17.2.0',
-                                  description: 'Does foo',
-                                  target_operating_systems: ['Linux'],
-                                  vendor: {
-                                    name: 'bob',
-                                    url: 'https://bob.example.com'}
-                                },
```

Comment:
I like the way the tests are written earlier and don;t see a need to change it. 
All we would want to test is a serialisation of given a PluginInfo object, with these changes the serialisation logic is tested in multiple places which is not clear IMO.